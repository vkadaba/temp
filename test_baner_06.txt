x, y: bool;
{
    if (!x)
    {
        while (!y)
        {
            switch (* a b)
            {
                case 1: { if (= x y) { y = z; } }
                case 2: { while ( > g h ) { x = + + < i j > k m = m n; } }
            }
            c = * d a;
        }
    }
}
