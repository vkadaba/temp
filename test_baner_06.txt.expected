x: bool #
y: bool #
a: int #
b: int #
z: bool #
g, h: ? #
i, j: ? #
k, m, n: ? #
c: int #
d: int #
